code/game/objects/items/pillow.dm

```diff
@@ -113,12 +128,26 @@
 		icon_state = "pillow_[variation]_t"
 		inhand_icon_state = "pillow_t"
 
+/// Puts a brick inside the pillow, increasing it's damage
+/obj/item/pillow/proc/become_bricked()
+	bricked = TRUE
+	var/datum/component/two_handed/two_handed = GetComponent(/datum/component/two_handed)
+	if(two_handed)
+		two_handed.force_unwielded += 5
+		two_handed.force_wielded += 10
+		force = two_handed.force_unwielded
+	else
+		force += 5
```

Comment:
`/datum/component/two_handed` is `COMPONENT_DUPE_UNIQUE_PASSARGS` so you can just do `AddComponent` with your new force values

Reply:
fuck true